cmake/apple/readme.txt

```diff
@@ -5,12 +5,16 @@ Solarus is licensed under GNU General Public License v3.
 This OSX package contains:
 - Solarus (Solarus.app): A GUI that allows you to run a Solarus quest and
   choose your audio and video settings.
-- Solarus-run (Solarus-run.app): An executable file that directly runs
-  a Solarus quest.
-  Run it with the path of your quest as argument
-  (the directory containing data/ or data.solarus or data.solarus.zip).
+- Solarus-run (Solarus-run.app): A package you can use either to distribute a 
+  standalone quest on other OSX systems with non-developer environment, or to 
+  run the solarus-run binary through the console.
+  The bundle is made to run any quest located into Solarus-run.app/Contents/Resources,
+  just replace the sample quest by your own one.
```

Comment:
The sample quest should not be in this bundle. The job of solarus-run is to run the quest of the current directory or of the command line.